libs/datasets/statistical_areas.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+from typing import Mapping
+import pandas as pd
+
+from libs import pipeline
+from libs.datasets import timeseries
+from libs.datasets.timeseries import MultiRegionTimeseriesDataset
+from covidactnow.datapublic.common_fields import CommonFields
+from libs.datasets import dataset_utils
+
+
+CBSA_LIST_PATH = "data/census-msa/list1_2020.xls"
```

Comment:
it would be great if this was not an excel file + CBSA_COLUMN was maybe a common field?

Reply:
I don't think it is worth the complexity of introducing yet another intermediate representation to simplify what is currently 10 lines of code and unlikely to change for at least a few months.
I don't want access/dependencies on the CBSA_COLUMN to creep in all over our code so trying to keep it isolated to this module.